src/app/nada-consta/nada-consta-visualizar.component.ts

```
@@ -25,42 +24,37 @@ import {ProgressBarModule} from 'primeng/progressbar';
-  private readonly nadaConstaService = inject(NadaConstaService);
-  private readonly router = inject(Router);
-
-  alunoId: number | null = null;
+  id: number | null = null;
   resultado: NadaConsta | null = null;
   erro: string | null = null;
   carregando = false;
-  private readonly destroyed$ = new Subject<void>();
+  private destroyed$ = new Subject<void>();
+
+  constructor(private nadaConstaService: NadaConstaService) {}
```

ESLint

[error] Prefer using the inject() function over constructor parameter injection. Use Angular's migration schematic to automatically refactor: ng generate @angular/core:inject

(@angular-eslint/prefer-inject)